Добавление экрана профиля и экрана воды

pollykon opened the pull request.

pollykon committed:
Добавление экрана профиля и экрана воды

Grandschtien reviewed (commented):
Код пока не смотрел, есть моменты по ui именно: 
- Нет кнопки выхода из прилы (она должна быть небольшая справа вверху)
- На экране воды должны быть текстовые поля а не лейблы(насколько я понимаю их надо редактировать)
- Картинки на экране воды на белом фоне(экспортируй их как svg и добавь)
- Лейбл "Выпито сегодня"
- Проверь расстояние от лейбла на экране воды и профиля по фигме
- Не работает кнопка назад на экране воды
- Экран воды появляется снизу, а должен справа налево

pollykon committed:
Добавление кнопки выхода из профиля

Grandschtien committed:
Merge branch 'dev' into polina-task10

Grandschtien committed:
fixes after merge

pollykon committed:
Добавление навигации на кнопки на экранах воды и профиля

pollykon committed:
Удаление белого фона

Grandschtien reviewed (commented):
- Не уходит клавиатура экране воды
- Таблица на экране воды не поднимается при появлении клавиатуры
- Поправь констреинты на экране профиля. При переключении с экрана результатов сильно заметно, что съезжает вьюха
- Проверь по фигме размер таблицы на экране профиля и воды
- Пофикси кнопку назад на экране воды (она слишком высоко и сама иконка   пиксельная с тенью - так не надо)
- Исправь на белый цвет кнопки выхода и опять же посмотри где она должна находиться (для этого почитай про размеры NavigationController в HIG от Apple)
- Не чиститься UserDefaults при выходе из аккаунта 
- В экране воды клавиатура с цифрами должна быть
- Текст для процентов посмотри каким шрифтом сделан в фигме
- Все надо переделать под Ванин класс upgradientpanel(по поводу верхних вью) - для этого проконсультируйся с ним он это делал и шарит
- Размеры шрифтов везде проверь
- Зачем у лейбла процентов ширина и высота в воде?
- Нет logout из файрбейза на кнопке exit
- Проверь отступы в таблице профиля между ячейками таблицы

pollykon committed:
Исправление клавиатуры

pollykon committed:
Исправление ошибок, связанных с размером

pollykon committed:
Исправление расположения вьюхи на экране профиля

Grandschtien reviewed (commented):
Осталось крч 4 момента по UI: 
- на экране воды навигейшн бар (где кнопка) не должен скролиться. Плюс заблокируй скрол, когда нет клавиатуры
- Также на экране воды надо сделать так, чтобы вьюшка поднималась при появлении клавиатуры 
- На экране профиля, да и воды тоже, проверить отступы между ячейками в коллекции
- Сделай белой кнопку выхода и спусти ее пониже чуток

pollykon committed:
Поднятие вью при появлении клавиатуры

Grandschtien commented on reshape/reshape/ProfileScreen/ProfileScreenInteractor.swift:
```diff
@@ -10,7 +10,19 @@ import Foundation
 
 final class ProfileScreenInteractor {
 	weak var output: ProfileScreenInteractorOutput?
+    let manager: ProfileManager
```
сделай приватным

Grandschtien commented on reshape/reshape/ProfileScreen/ProfileScreenViewController.swift:
```diff
@@ -2,30 +2,210 @@
+        return photo
+    }()
+
+    private let informHeaderLabel: UILabel = {
```
lazy var

Grandschtien commented on reshape/reshape/WaterScreen/WaterCells/WaterCollectionCell.swift:
```diff
@@ -0,0 +1,43 @@
+//  Created by Полина Константинова on 19.04.2022.
+//
+
+import UIKit
```
А зачем два файла одинаковых в папке Cells и WaterCells

Grandschtien reviewed (commented):
Поправь поднятие вьюшки с клавой. И еще я не стал указывать в коментах про то, что можно в целом подогнать вьюху из result screen к твоей вьюхе на профиле, так как это может быть долго, но в целом это можно сделать (по желанию крч)

pollykon committed:
no message

Grandschtien committed:
pod deintegrate

Grandschtien committed:
Merge remote-tracking branch 'origin/dev' into polina-task10

Grandschtien committed:
Замержил ветку дев

Есть 2 тудухи в роутере экрана профиля и на самом экране профиля, все описал в них
